bin/resources/GameIndex.yaml

```diff
@@ -787,6 +787,8 @@ SCAJ-20068:
     - SoftwareRendererFMVHack # Fixes brightness and overlapping subtitles.
   gsHWFixes:
     roundSprite: 1 # Fixes font artifacts.
+    skipDrawStart: 1 # Removes blur in background in battles.
+    skipDrawEnd: 1 # Removes blur in background in battles.
```

Comment:
Skipdraw should be removed, unless absolutely necessary with tc issues.
Same for the others.

Reply:
It's not breaking other effects, I tested it extensively.

Reply:
The blur is intended , people can make a per game setting if someone hates it

Reply:
GameIndex with skipdraw:

    skipDrawStart: 1 # Removes blur in background in battles.
    skipDrawEnd: 1 # Removes blur in background in battles.
[GameIndex.zip](https://github.com/PCSX2/pcsx2/files/12178413/GameIndex.zip)

I'll remove it for now, photos are still in a mess but I can fix that later even if PR is merged.